cli/flox/doc/flox-activate.md

```diff
@@ -10,99 +10,102 @@ flox-activate - activate environments
+     [-d=<path> | -r=<owner/name>]
+     [ -t ]
+     [ -- <command> [ <arguments> ] ]
+```
 
 # DESCRIPTION
 
-Sets environment variables and aliases, runs hooks and adds environment
-`bin` directories to your `$PATH`. Can be invoked from an interactive
-terminal to launch a sub-shell, non-interactively to produce
-a series of commands to be sourced by your current `$SHELL`,
-or with a command and arguments to be invoked directly.
+Sets environment variables and aliases,
+runs hooks,
+and adds `bin` directories to your `$PATH`.
 
+Launches an interactive subshell when invoked as `flox activate` from an
+interactive shell.
+Launches a subshell non-interactively when invoked with a command and
+arguments.
+May also be invoked as `$(flox activate)` to produce commands to be sourced
+by your current `$SHELL`.
 
+When invoked interactively,
+the shell prompt will be modified to display the active environments,
+as shown below:
+```
+flox [env1 env2 env3] <normal prompt>
+```
 
 # OPTIONS
 
-```{.include}
-./include/general-options.md
-./include/environment-options.md
-```
-
 ## Activate Options
 
-[ -- `<command>` [ `<argument>` ] ]
+`-- <command> [ <arguments> ]`
 :   Command to run in the environment.
     Spawns the command in a subshell
     that does not leak into the calling process.
 
+`-t`, `--trust`
+:   Trust a remote environment for this activation.
 
-# ENVIRONMENT VARIABLES
-
-`$FLOX_ENV`
-:   Absolute path to the _install prefix_ of the environment being activated.
-
-    Set by `flox activate` before executing `shell.hook`.
-
-    If multiple environments are indicated, each `shell.hook` will run with
-    its associated `FLOX_ENV` set properly, and the activated environment
-    will have `FLOX_ENV` set to the first environment indicated on the CLI.
-
-    This variable may be used to set other environment variables such as
-    `MANPATH`, `PKG_CONFIG_PATH`, `PYTHONPATH`, etc so that relevant tooling
-    will search these directories to locate files and resources from
-    the environment.
-
-    **N.B.** the default shell hook for newly-created environments will
-    source the `$FLOX_ENV/etc/profile` file at activation if it exists.
-    This behavior can be viewed/modified with `flox edit`.
+```{.include}
+./include/environment-options.md
+./include/general-options.md
+```
 
-## Language packs _(**experimental**)_
+# ENVIRONMENT VARIABLES
 
-Language packs help you develop with flox the way **you** work, making it
-possible to install and use compilers, interpreters, libraries and modules
-in much the way you would on any other operating system.
+## `$FLOX_ENV`
+The absolute path to the _install prefix_ of the environment being activated.
+Set by `flox activate` before executing `shell.hook`.
+
+If multiple environments are active,
+each `shell.hook` will run with its associated `FLOX_ENV` set properly,
+and the activated environment will have `FLOX_ENV` set to the left-most
+environment listed in the prompt.
```

Comment:
This sentence reads a bit weirdly.
It also sounds like we set the variable to `<env name>` rather than a PATH.


There is also no `shell.hook` I guess you mean `hook.script`?

Suggestion: 

```
Upon activation, `FLOX_ENV` is set to the environment's respective install prefix
before an it's shell hook (`hook.script`) is called.
If `FLOX_ENV` is already set from another active environment,
it will be reset according to the latest activation.
```

Reply:
I've just deleted this environment variable from the manpage. I don't see a reason a user needs to know any of this.